database/migrations/20220720171428-create-news.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('News', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      name: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      content: {
+        allowNull: false,
+        type: Sequelize.TEXT,
+      },
+      image: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      categoryId: {
```

Comment:
pregunta: la clave foránea es realmente nullable?

Reply:
@gabodmc Gabo, a nivel negocio, una noticia siempre tiene que tener una categoría?

Reply:
@ghnoob  depende si es necesario que exista la entidad a la cual se relaciona...

Reply:
> @gabodmc Gabo, a nivel negocio, una noticia siempre tiene que tener una categoría?

Yo tendría entendido que si, ya que si no como aplicaríamos un filter sobre la misma

Reply:
En este caso no esta solicitado, asi que no es mandatorio